### ISSUE TITLE
Catch-up points

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/Tutorials#107]
to get everyone on same track if we only teach a single topic (e.g., metabolomics)

**Comment by oliveralka** [comment]
Should we include here as well a 5-minute hands-on presentation e.g. for ToppView? 
What can you see and explore when the workflow finished?

**Comment by timosachsenberg** [comment]
Yep that’s what I recently did. Not sure if it needs to be part of the handout - though we could think about adding a list: you should have learned following stuff... and then use this as a guide to recap things in a catch-up
